cobbler/cli.py

```diff
@@ -317,10 +317,10 @@ def __init__(self, cliargs):
         self.args = cliargs
 
     def start_task(self, name, options):
-        """
+        r"""
```

Comment:
Is this a typo or is "functional" somehow?

Reply:
This is called a "raw docstring" and is intentional since I added a backslash as escaping character in line 323.
> Use r"""raw triple double quotes""" if you use any backslashes in your docstrings

 See https://www.python.org/dev/peps/pep-0257

Reply:
Thanks for clearing that up. I did not know.